test/run.test.js

```diff
@@ -697,3 +697,87 @@ test('should count resets', t => {
     t.end()
   })
 })
+
+test('should parse HAR file', (t) => {
+  t.plan(6)
+
+  run({
+    url: 'https://httpbin.org',
```

Comment:
Are we calling httpbin directly? Maybe use an internal server?

Reply:
Yes we are.

The difficulty with using an interval server is to change the har file.
We'll need to change the urls in file to match the internal server, which port is dynamic. 
It can be done by copying the fixture file, altering the copy, use the copy during test, and remove it.

I though it was ok to use httpbin, given it's meant for testing. If you prefer, I can change that in favor of the solution above.

Reply:
Please do the above. Touching the network make the test not pass in case of an outage.